main.py

```diff
@@ -39,12 +38,13 @@ def get_players():
         players = [(data['members'][member]['name'],
                     data['members'][member]['local_score'],
                     data['members'][member]['stars'],
-                    int(data['members'][member]['last_star_ts'])) for member in data['members']]
+                    int(data['members'][member]['last_star_ts']),
+                    data['members'][member]['completion_day_level']) for member in data['members']] ##This is the data for all the days
```

Comment:
You'll also have to add a type hint on line 152 to bring the day in as an integer

Reply:
That data is a dictionary not a string.

Reply:
Unless I am missing something

Reply:
My objection here is that the keys for the dictionary should really be an integer (so 1 and 2 rather than '1' and '2'), which if they were, it would make life way simpler elsewhere in the code
In fact, really this shouldn't be a dict, it should just be a list (there's no point in having keys when the key is just the index+1)
Basically then, if you could maybe replace the dictionary with a list (possibly with something like this: `[star for star in data['members'][member]['completion_day_level']]`, if you do this, it could probably do with a comment) and then refactor the code below to use the integer indexes instead of the string keys

Reply:
I'll change this in a future commit, closing for now